README.md

```diff
@@ -22,7 +22,11 @@ Or install it yourself as:
 
 ```ruby
 
-summary_document = TelephoneAppointments::SummaryDocumentActivity.new(appointment_id: id, owner_uid: uid)
+summary_document = TelephoneAppointments::SummaryDocumentActivity.new(
+  appointment_id: id,
+  owner_uid: uid, 
+  message: message # 'postal' or 'digital'
```

Comment:
I don't think striving to match up arguments internally with TAP is something we should be focused on. The API is an external representation of the system, it doesn't need to honour internal behaviour / naming necessarily.

Another issue here is what happens if a wayward client specifies `message: 'welpwelpwelp'` or something? Perhaps reducing this to a boolean, or at least performing some validation on the `message` (after renaming it to something more inline with what we're describing 😉 ) makes more sense here.